Project/Assets/Editor/CustomInspectors/TwineNodeEditor.cs

```diff
@@ -16,12 +16,12 @@ public override void OnInspectorGUI ()
 
 		node.isDecisionNode = EditorGUILayout.Toggle ("Decision node?", node.isDecisionNode);
 		node.objectsToTrigger = PrairieGUI.drawObjectList ("Objects To Trigger", node.objectsToTrigger);
-		node.name = EditorGUILayout.TextField ("Name", node.name);
-		node.tags = PrairieGUI.drawPrimitiveList ("Tags", node.tags);
+		node.name = PrairieGUI.TextFieldReadOnly ("Name", node.name);
+		node.tags = PrairieGUI.drawPrimitiveListReadOnly ("Tags", node.tags);
```

Comment:
I don't think there are many compelling reasons to include `tags` as part of the GUI. I can't imagine a situation in which a user would need to consult the tags array on a `TwineNode` after import.